app/templates/administrative-units/administrative-unit/sites/index.hbs

```diff
@@ -25,9 +25,15 @@
       <th></th>
     </c.header>
     <c.body as |site|>
-      <td><AuLink @route="administrative-units.administrative-unit.sites">{{site.address.fullAddress}}</AuLink></td>
+      <td>
+        <AuLink @model={{site.id}} @route="administrative-units.administrative-unit.sites.site.index">
```

Comment:
Nitpicky: It's usually better to not link to an index route directly. That's an implementation detail of that route. The parent route doesn't need to know that `administrative-units.administrative-unit.sites.site` uses an index route.

I would change this to `@route="administrative-units.administrative-unit.sites.site"`